soft4pes/model/grid/rl_grid.py

```diff
@@ -24,6 +26,8 @@ class RLGrid:
         Inductance [H].
     base : base value object
         Base values.
+    ig_ref_init : 1 x 2 ndarray of floats, optional
+        Reference at time step kTs = 0 for starting simulation from steady state.
```

Comment:
k, k+1, k+2, ... --> discrete time steps
k T_s, (k+1) T_s, (k+2) T_s, ... --> discrete-time instants

Reply:
Resolved across the repository